Gemfile

```diff
@@ -26,8 +26,7 @@ gem 'git', '~> 1.2.5'
 gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
-gem 'aweplug', github: 'awestruct/aweplug'
```

Comment:
Don't delete this line, just comment it out. It makes it harder to revert back to the master branch if we need to go look up the exact syntax.